src/ai/backend/gateway/vfolder.py

```diff
@@ -178,6 +201,8 @@ def get_folder_hostpath(row: VFolderRow, app):
     t.Dict({
         t.Key('name'): tx.Slug(allow_dot=True),
         t.Key('host', default=None) >> 'folder_host': t.String | t.Null,
+        t.Key('usage_mode', default='general'): t.String | t.Null,
+        t.Key('permission', default='rw'): t.String | t.Null,
```

Comment:
How about using `tx.Enum` (`tx` is an aliased import of `ai.backend.common.validators`) to validate earlier?

Reply:
Fixed!